lib/checks/keyboard/focusable-element.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Note:
+ * Check
+ * - if element is focusable
+ * - if element is in focus order via `tabindex`
+ */
+const isFocusable = virtualNode.isFocusable;
+
+let tabIndex = virtualNode.actualNode.getAttribute('tabindex');
```

Comment:
Just a minor suggestion: If you `parseInt` when you access the attribute then you shouldn't have to do it twice in the ternary.